flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/FlinkRelNode.scala

```diff
@@ -32,6 +34,21 @@ import scala.collection.JavaConversions._
   */
 trait FlinkRelNode extends RelNode {
 
+  /**
+    * Returns a string which describes the relational expression with attributes which
+    * contribute to the plan output.
+    *
+    * This method leverages [[RelNode#explain]] with
+    * [[org.apache.calcite.sql.SqlExplainLevel.EXPPLAN_ATTRIBUTES]] explain level.
+    */
+  def getRelDescription: String = {
```

Comment:
oops... i find `AbstractRelNode` also has a method named `getDescription`. It's better to use another name...

Reply:
After discussed with @godfreyhe , we reached a consensus to pick `getRelDetailedDescription` as the method name.